CountdownWidget/CountdownWidget/src/countdownWidget.ts

```diff
@@ -37,174 +43,174 @@ export class CountdownWiget {
 		return this.showCountdown(widgetSettings);
 	}
 
-	private showSprintWidget(customSettings: ISettings, workingdays: System_Contracts.DayOfWeek[]) {
-		const webContext = VSS.getWebContext();
-		const teamContext: TFS_Core_Contracts.TeamContext = {
-			project: "",
-			projectId: webContext.project.id,
-			team: "",
-			teamId: webContext.team.id,
-		};
-		const workClient: Work_Client.WorkHttpClient = Service.VssConnection
-			.getConnection()
-			.getHttpClient(Work_Client.WorkHttpClient, WebApi_Constants.ServiceInstanceTypes.TFS);
+	private showCountdown(widgetSettings) {
+		try {
+			this.parseSettings(widgetSettings);
+
+			this.fetchWorkingDays()
+				.then(() => {
+					return this.determineRenderValues();
+				})
+				.then(() => {
+					this.calculateCountdown();
+					this.calculateBackgroundColor();
+					this.renderWidget();
+				});
+
+			return this.WidgetHelpers.WidgetStatusHelper.Success();
+		} catch (e) {
+			// Satisfy the linter
```

Comment:
Any way to capture a log line here? Something we could use to troubleshoot if necessary? It seems a bit counter-intuitive to just swallow a potential problem and not capture something about it.